go/consensus/tendermint/light/client.go

```diff
@@ -228,6 +122,7 @@ func NewClient(ctx context.Context, cfg ClientConfig) (Client, error) {
 		tmlightdb.New(tmdb.NewMemDB(), ""), // TODO: Make the database configurable.
```

Comment:
While we're here, given that we will be using the light client more, should we make this persistent (e.g. in the tendermint part of the data directory, where the location is passed via `cfg.LightDbPath` or similar)? I guess it makes sense in case there will only ever be a single light client instance per node (which probably makes sense).

Reply:
Yeah I think this makes sense.

Reply:
As discussed, there's now two light clients. The "internal" one, only used for state sync, still uses the in-memory store.
The Light client service (a light client that is exposed to other backends as well) is persistent and is initialized with trusted blocks from the locally synced tendermint backend.